### ISSUE TITLE
Why do I always encounter CUDA out of memory problem when I load the load_model_process function? Can the RTX 3090 be used for the BLIP-2 model?"

### ISSUE THREAD

**Issue by zhangmenghuan-mh** [salesforce/LAVIS#670]
Why do I always encounter CUDA out of memory problem when I load the load_model_process function? Can the RTX 3090 be used for the BLIP-2 model?"

**Comment by Thomas2419** [comment]
Yes, I'm using mine for that right now.  More information may be needed to help, I know transformers version is an extremely common issue plaguing people.  Specifically I'm using it for image captioning though so perhaps some nuance on your task may be in play here.

**Comment by shams2023** [comment]
> 是的，我现在正在用我的。可能需要更多信息来帮助，我知道变形金刚版本是困扰人们的一个极其常见的问题。具体来说，我将它用于图像字幕，所以也许您的任务中可能存在一些细微差别。

What is the image captioning ability of Blip2 for you? Can it describe the content and details of the image well?

**Comment by Thomas2419** [comment]
Well I have trained a model on a set of labels, they're complicated and interconnected so multilabel classifiers have failed to effectively do this, so i use labels as my captions in my finetuning to do it. So I'm unsure of the performance of the base model. As far as I've seen the blip2 opt2.7b model is quite exceptional, it even scores quite high on benchmarks, I often look at papers with code website and on their image captioning benchmark it lands at number 4, which is quite impressive when it's competing with mPLUG OFA and GIT, which if i remember correctly are all significantly larger.

**Comment by shams2023** [comment]
> 好吧，我已经在一组标签上训练了一个模型，它们很复杂且相互关联，因此多标签分类器无法有效地做到这一点，所以我在微调中使用标签作为标题来做到这一点。所以我不确定基本模型的性能。据我所知，blip2 opt2.7b 模型非常出色，它甚至在基准测试中得分相当高，我经常查看带有代码网站的论文，在他们的图像标题基准测试中，它排在第 4 位，这在它与 mPLUG OFA 和 GIT 竞争时令人印象深刻，如果我没记错的话，它们都明显更大。

I saw your answer, thank you very much for answering my question! The problem I am currently facing is how to use my own dataset to fine tune BLIP2. Have you succeeded? If successful, can you share the code?
I am very much looking forward to your reply, as it will be of great help to me!
Thank you!
